etc/schema.json

```diff
@@ -6866,6 +6866,10 @@
                 }
             },
             "additionalProperties": false
+        },
+        "ndpi": {
```

Comment:
Same question than for doc : does this plugin stuff belong in base suricata etc schema.json ? 
Can a plugin propose its json schema extension ?

Reply:
Its only here so we can run S-V with the ndpi plugin loaded, which creates an ndpi structure. I don't have a better way to "extend" the schema for now.

I guess the question is, if we ship the plugin, should add it to the schema as well.  I don't actually know the full schema tho, thats not my work.

Reply:
> Could the main schema "import" a plugins' schema/ folder? Would that make sense/ be better?

Not really, well not without some custom tooling to merge the base schema with this extension then validating. But thats a whole other project.

Reply:
Added to description that the contents is provided by a 3rd party library.